docs/reference/search/profile.asciidoc

```diff
@@ -166,9 +166,16 @@ The API returns the following result:
             "rewrite_time": 451233,
             "collector": [
               {
-                "name": "SimpleTopScoreDocCollector",
-                "reason": "search_top_hits",
-                "time_in_nanos": 775274
+                "name": "QueryPhaseCollector",
```

Comment:
Are there BWC concerns as we are changing the output of the profiler?

Reply:
No. We break the profiler output every time we change the way query execution runs internally, e.g. we broke it when we implemented the filter-by-filter optimization. It's expected that internal implementation details may change.

Reply:
sorry for the additional response that did not add value, I saw no other previous replies when I posted mine.